defaults/main.yml

```diff
@@ -50,6 +50,11 @@ gitea_db_backup_frequency: 'daily'
 gitea_db_backup_timeout: 300
 gitea_db_backup_user: 'root'
 
+# Init
+#gitea_init_orgs:
+#  - org1, 
+#  - org2
```

Comment:
If it's a list we can just define it as empty by default:
```suggestion
#gitea_init_orgs: []
#  - org1
#  - org2
```
Also, unnecessary comma.